### ISSUE TITLE
loss function of cox model

### ISSUE THREAD

**Issue by EQUIWDH** [pbreheny/ncvreg#42]
Hello, I am wondering if `fit$loss` returned by `ncvsurv` represents only the cox -2*logL calculating by estImated $\beta$ or the -2logL + penalty form ，thanks!

**Comment by EQUIWDH** [comment]
Sorry for disturbing, I am use the loss function structure provided by `loss.R` which is
```
loss<- function(x,beta,y ,total=TRUE) {
  eta = x%*%beta
  ind <- order(y[,1])
  d <- as.double(y[ind,2])
  if (is.matrix(eta)) {
    eta <- eta[ind, , drop=FALSE]
    r <- apply(eta, 2, function(x) rev(cumsum(rev(exp(x)))))
  } else {
    eta <- eta[ind]
    r <- rev(cumsum(rev(exp(eta))))
  }
  if (total) {
    return(-2*(crossprod(d, eta) - crossprod(d, log(r))))
  } else { 
    return(-2*(eta[d==1,] - log(r)[d==1,]))
  }
}
```
compare`loss(fit$X,fit$beta,train$y)` and `fit$loss`, I find they don't correspond as the picture showing

**Comment by EQUIWDH** [comment]
![test](https://user-images.githubusercontent.com/101512154/233275067-bbb4fcc4-7352-40c1-9cf4-21b27fe14bbe.png)
